modules/mobile/spec/support/schemas/letter_beneficiary.json

```diff
@@ -8,7 +8,7 @@
         "attributes": {
           "properties": {
             "benefitInformation": {
-              "oneOf": [
+              "anyOf": [
```

Comment:
EVSS was omitting some boolean field based on if it was a veteran or dependent. the schema was ensuring it was one or the other. Lighthouse includes all fields and just turns them false if not relevant.